.env.sample

```
@@ -9,7 +9,34 @@ BROWSE_INCREMENT="10"
 # "false" to revert filterName to the legacy name-substring regex without a
 # code revert. Any other value (or unset) is enabled.
 # SEARCH_V2_ENABLED="false"
+# The host answer: running the app straight on the host, the database and
+# Mailpit are localhost on the published MONGO_PORT / MAILPIT_SMTP_PORT. In
+# the dev container they are compose services, and
+# .devcontainer/docker-compose.yml sets DB_URI and SMTP_HOST to those service
+# names on the container itself — so a plain `cp .env.sample .env` is correct
+# for both paths and neither needs editing.
 DB_URI="mongodb://localhost:27017/blueprintnotincluded"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)